helm-chart/renku/values.yaml

```diff
@@ -126,6 +136,26 @@ global:
       tag: "0.0.2"
     customCAs: []
       # - secret:
+  
+  ## Values for the RabbitMQ setup.
+  rabbitmq:
+    ## Whether to install RabbitMQ as part of the Renku helm chart.
+    ## If set to false, then an external RabbitMQ instance needs to be available
+    ## and the external section below should be filled out.
+    enabled: true
```

Comment:
Should this not be set to false? Afaict, no Renku services use Rabbitmq yet. So we'd just be forcing everyone to run rabbit for no reason.